packages/elements/src/tree/elements/tree.ts

```diff
@@ -425,8 +425,34 @@ export class Tree<T extends TreeDataItem = TreeDataItem> extends List<T> {
       return this.composer.getItemPropertyValue(item, 'value') as string || '';
     });
   }
-  public set values (value: string[]) {
-    super.values = value;
+  public set values (values: string[]) {
+    if (!Array.isArray(values)) {
```

Comment:
Why need to check Array.isArray?  values type only accept array string already, isn't it?

Reply:
It not related to the type definition. User may set values = 'some string' and we should notify an error message.